gits/terminal.py

```diff
@@ -402,8 +402,18 @@ def _cap_dim(self):
         pass
 
     def _cap_dl(self, n):
-        """Deletes ``n`` number of lines in their entirety, with the lines
-        below moving up to fill the gap that is left.
+        """Deletes ``n`` number of lines.
+
+        On the one hand, the specification says that the dl capability should
+        delete ``n`` number of lines, on the other hand, in the reality, dl
+        just scrolls up ``n`` number of lines. Notice, that dl should work
+        together with another capability that will put the cursor on the
+        line that is going to be deleted.
+
+        For example, in tests dl works together with the home capability, but
+        it doesn't mean that the capabilities are always used together.
+        Moreover, it's quite difficult to find a program which uses dl. As far
+        as we know, vi, nano, top, htop, mc don't use it.
```

Comment:
The docstring is getting more and more beautiful. It does my heart good to see it.
A couple of additional comments: 
* I suggest we combine the last two paragraphs into one, because they are closely connected with each other. The last paragraph starts with the words "For example", so obviously it belongs to the previous paragraph.
* I suggest we delete `Moreover, it's quite difficult to find a program which uses dl. As far as we know, vi, nano, top, htop, mc don't use it.` because it's not connected with the whole text. In the text we try to explain how `dl` works and, in my opinion, we did our best. How difficult to find any example of using `dl` is a different story.